lib/index.js

```diff
@@ -1,4 +1,4 @@
-const jsonClient = require('@cuvva/json-client');
```

Comment:
we need the dependency to be the `@cuvva` one for the request-id tracing

Reply:
Ah merge conflict oops